### PR TITLE
[zh-CN]: sync&add translation for `PictureInPictureEvent`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the https://github.com/mdn/translated-content/issues/23557

~depends on https://github.com/mdn/content/pull/35887~

### CODE REVIEW

File: files/zh-cn/web/api/pictureinpictureevent/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: PictureInPictureEvent
+slug: Web/API/PictureInPictureEvent
+l10n:
+  sourceCommit: 95dff5ec1195f072b8e48a2273294933670b1e99
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/pictureinpictureevent/index.md:5]:
```suggestion
  sourceCommit: bb4d21e3c6e71db4e0ba983a450d6ed628e82670
```

File: files/zh-cn/web/css/_colon_picture-in-picture/index.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 title: ":picture-in-picture"
 slug: Web/CSS/:picture-in-picture
+l10n:
+  sourceCommit: 04ebe57066db2cff350018649bdb15b2a10c67ba
 ---
 
 {{CSSRef}}
```
Comment (yin1999) [inline files/zh-cn/web/css/_colon_picture-in-picture/index.md:8]: 下面的示例也麻烦翻译一下

File: files/zh-cn/web/css/_colon_picture-in-picture/index.md [hunk 3]
```diff
@@ -28,11 +30,11 @@ slug: Web/CSS/:picture-in-picture
...
 
 ```html
-<h1>MDN Web Docs Demo: :picture-in-picture pseudo-class</h1>
+<h1>MDN Web Docs 演示：:picture-in-picture 伪类</h1>
 
 <p>
-  This demo uses the <code>:picture-in-picture</code> pseudo-class to
-  automatically change the style of a video entirely using CSS.
+  此演示使用 <code>:picture-in-picture</code> 伪类通过 CSS
+  自动完全改变视频的样式。
```
Comment (skyclouds2001) [inline files/zh-cn/web/css/_colon_picture-in-picture/index.md:37]:
```suggestion
  此演示使用 <code>:picture-in-picture</code> 伪类
  通过 CSS 自动完全改变视频的样式。
```

Comment (yin1999) [inline files/zh-cn/web/css/_colon_picture-in-picture/index.md:37]:
```suggestion
  此演示使用 CSS <code>:picture-in-picture</code>
  伪类自动改变视频的完整样式。
```

**Review by yin1999 — COMMENTED** [review]
这样翻译如何：

**Review by yin1999 — APPROVED** [review]
Thank you 🙏

**Pull request merged**